### ISSUE TITLE
Compilation problem

### ISSUE THREAD

**Issue by mlotysz** [VanDavv/Pat2016_Lukasz_Pilatowski#1]
`mvn clean test`

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project Zadanie1: Compilation failure
[ERROR] /home/mlo/work/patronage/zad1_lpilatowski/src/main/java/resources/MovieResource.java:[32,44] diamond operator is not supported in -source 1.5
[ERROR] (use -source 7 or higher to enable diamond operator)
```

My env:

```
> mvn -v
Picked up _JAVA_OPTIONS: -Dawt.useSystemAAFontSettings=lcd
Apache Maven 3.3.9 (NON-CANONICAL_2015-11-23T13:17:27+03:00_root; 2015-11-23T11:17:27+01:00)
Maven home: /opt/maven
Java version: 1.8.0_66, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-8-jdk/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "4.2.5-1-arch", arch: "amd64", family: "unix"
```

**Comment by VanDavv** [comment]
Should be fine now

**Comment by mlotysz** [comment]
Fixed by 8e9e54f
